code/modules/clothing/spacesuits/rig/suits/ert.dm

```diff
@@ -51,46 +53,51 @@
 	desc = "A suit worn by the medical division of an Emergency Response Team. Has white highlights. Armoured and space ready."
 	suit_type = "ERT medic"
 	icon_state = "ert_medical_rig"
-	emp_protection = 30
+	armor = list(melee = 50, bullet = 40, laser = 20, energy = 15, bomb = 30, bio = 100, rad = 100)
+	slowdown = 0
+	emp_protection = 20
 
 	initial_modules = list(
 		/obj/item/rig_module/ai_container,
 		/obj/item/rig_module/vision/medhud,
 		/obj/item/rig_module/maneuvering_jets,
 		/obj/item/rig_module/device/healthscanner,
 		/obj/item/rig_module/chem_dispenser/injector,
-		/obj/item/rig_module/actuators
+		/obj/item/rig_module/actuators/combat
```

Comment:
No.
ERT definitly shouldnt get combat leg actuators.
They allow you to leap towards a target and acquire a aggressive grab of the target.